example-storage/README.md

```diff
@@ -0,0 +1,114 @@
+# 🖼️ Images With React
+
+A simple image sharing app built with Appwrite and React, demonstrating [Appwrite Storage](https://appwrite.io/docs/storage) and [Permissions](https://appwrite.io/docs/permissions).
+
+<img width="466" alt="image" src="https://user-images.githubusercontent.com/723517/195679601-9dbfc7e5-37d8-4144-84e6-a103ad85979f.png">
+
+Adapted from [Todo With React](https://github.com/appwrite/demo-todo-with-react).
+
+## 🎬 Getting Started
+
+### 🤘 Install Appwrite 
+Follow our simple [Installation Guide](https://appwrite.io/docs/installation) to get Appwrite up and running in no time. You can either deploy Appwrite on your local machine or, on any cloud provider of your choice. 
+
+> Note: If you setup Appwrite on your local machine, you will need to create a public IP so that your hosted frontend can access it.
+  
+We need to make a few configuration changes to your Appwrite server. 
+
+1. (Optional) Configure a Storage in Appwrite
+
+    Although you can deploy the app with the default Local storage, this app is intended to allow you to experiment with other Appwrite Storage adapters. For example, you could [configure Backblaze B2 Cloud Storage to store files in the cloud](https://dev.to/appwrite/appwrite-storage-meets-backblaze-b2-4m2e).
+    
+    If you do configure a Storage, remember to restart Appwrite after editing your `.env` file. This can be easily done by running the following command.
+    
+    ```bash
+    docker-compose up -d
+    ```
+
+2. Create a new project in the Appwrite console.
+
+3. Add a new Web App to the project and enter your website's endpoint (`localhost, <project-name>.vercel.app etc`)
+
+    ![Create Web App](https://user-images.githubusercontent.com/20852629/113019434-3c27c900-919f-11eb-997c-1da5a8303ceb.png)
+    
+4. Click **Storage** in the left navigation menu to create a new bucket.
+  * Give the bucket a suitable name, for example, `user-images`, and click **Create**.
+  * Make a note of the **Bucket ID** on the right of the Settings page.
+  * Scroll down the Settings page, and click **Add Role**. 
+  * In the **Role** dropdown, select `users`, and enable the **Create** permission.
+  * Enable **File Security**.
+  * Click **Update** to update the bucket's settings.
+
+    <img width="642" alt="Bucket Settings" src="https://user-images.githubusercontent.com/723517/195677596-f52fd8d0-c94c-4670-9bfb-46565e9b899f.png">
+
+## 🚀 Deploy the Front End
+
+You have two options to deploy the front-end and we will cover both of them here. In either case, you will need to fill in these environment variables that help your frontend connect to Appwrite.
```

Comment:
For local copy the `.env.example` file to `.env`.

Reply:
I say this lower down, in the **Run Locally** section. (This was copied verbatim from https://github.com/appwrite/demo-todo-with-react)